src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.DukeList;
+import duke.UserInterface;
+
+/**
+ * Represents a command to list all tasks in a dukelist.
+ */
+public class ListCommand implements Command {
+    private final DukeList dukeList;
+    private final UserInterface ui;
+
+    /**
+     * Creates a command that lists tasks in the given dukelist onto the specified
+     * ui.
+     *
+     * @param dukeList of which tasks
```

Comment:
might be better if you write @param dukeList User's todo list (Start the description with a capitalized letter). Remember to put a punctuation after each parameter description :)

Reply:
It seems like capitalising the first letter of the description isn't a common practice in general, if I were to direct you the java's String class's [documentation](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html). Again, I may be wrong and that it is part of the module's guideline. Thanks anyways!

Reply:
On the point of having punctuation, I'm also not sure if it is a [convention](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Map.html) or a standard in the module. It does make sense that it looks neater that way, however.